core/src/env_platform/db_handler.bal

```diff
@@ -0,0 +1,79 @@
+import ballerina/config as conf;
+import ballerina/mongodb;
+
+// Mongodb configurations.
+mongodb:ClientConfig mongoConfig = {
+    host: conf:getAsString("DB_HOST"),
+    username: conf:getAsString("DB_USER_NAME"),
+    password: conf:getAsString("DB_PASSWORD"),
+    options: {sslEnabled: false, serverSelectionTimeout: 500}
+};
+mongodb:Client mongoClient = check new (mongoConfig);
+mongodb:Database mongoDatabase = check mongoClient->getDatabase("EnvironmentPlatform");
+mongodb:Collection applicationCollection = check mongoDatabase->getCollection("applications");
+
+# The `postApplicationToDB` function will post the application to the applications collection in the database
+# 
+# + form - The TreeRemocalForm Type record is accepted.
+# 
+# + return - This function will return null if application is added to the database or else return mongodb:Database error
+function postApplicationToDB(TreeRemovalForm form) returns error? {
+
+    json[] locations = [];
+    foreach Location location in form.area {
+        locations.push(<json>{"Latitude": location.latitude, "longitude": location.longitude});
+    }
```

Comment:
@yashodgayashan shall we separate these tasks with new lines and add comments like `Make the location json` and etc.. This will make it more readable.

Reply:
I will do it or I can make separate functions as mentioned above.

Reply:
@sanjulamadurapperuma making separate function would be better right? we can then reuse if it will be needed again.

Reply:
yes

Reply:
> @yashodgayashan shall we separate these tasks with new lines and add comments like `Make the location json` and etc.. This will make it more readable.

@dehami resolved requested changes